### ISSUE TITLE
P.291 first paragraph

### ISSUE THREAD

**Issue by hongsoog** [rasbt/machine-learning-book#70]
In original text of the first paragraph in  p. 291
> We can see that the MSE on the training dataset is **larger**  than on the test set, which is an indicator that our model is slightly overfitting the training data in this case. 

should be corrected as follows:
> We can see that the MSE on the training dataset is **less**  than on the test set, which is an indicator that our model is slightly overfitting the training data in this case.

**Comment by rasbt** [comment]
Good catch, thanks!
